src/index.html

```diff
@@ -1,13 +1,62 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/style.css">
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <img
+        class="product-card__image"
+        src="./images/imac.jpeg"
+        alt="product-image"
+      >
+
+      <p class="product-card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+
+      <p class="product-card__code">
+        Product code: 195434
+      </p>
+
+      <div class="product-card__review">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="product-card__review product-card__review--text">
+          Reviews: 5
+        </p>
+      </div>
+
+      <div class="product-card__price">
+        <p class="product-card__price product-card__price--text">
+          Price:
+        </p>
+
+        <p class="product-card__price product-card__price--value">
+          $2,199
+        </p>
```

Comment:
no need to use modifiers for elements review and price, you can just use elements
```suggestion
      <div class="product-card__review">
        <div class="stars stars--4">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>

        <p class="product-card__review-text">
          Reviews: 5
        </p>
      </div>

      <div class="product-card__price">
        <p class="product-card__price-text">
          Price:
        </p>

        <p class="product-card__price-value">
          $2,199
        </p>
```